src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bs`        | Ball-and-Stick(s) model \[[Behrens2003](#behrens2003)\],\[[Behrens2007](#behrens2007)\],\[[Jbabdi2012](#jbabdi2012)\]                           | One [spherical coordinates](#data-spherical) image with parameter name "`sticks`", providing both fibre volume fractions and orientations using polar angles;<br>Optional scalar images with parameter names {"`bzero`", "`dmean`", "`dstd`"} providing the model-estimated *b*=0 signal intensity, mean stick diffusivity, and standard deviation of stick diffusivities respectively                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |

```

Comment:
Good choice of acronym!

Reply:
:laughing: 

I've still not gotten this to the point where I'm happy with it. Really, what we want to be encoding here is not specifically "ball" and "sticks", but the most general case of discrete mixture models that use different mathematical models for the different components (e.g. zeppelin(s), rackets), of which FSL bedpostx output / "ball-and-sticks" are merely specific instances of such. But I've not actually used such models, I'm only approaching this from a theoretical perspective. Would be interested in interacting with FMRIB or anyone who has experimented with the breadth of this class of discrete mixture models.